### PR TITLE
build(comp): Component DropListSV to make menu lesson for student

### PR DESCRIPTION
**Pull request by Binkeurss**
Component DropListSV to make menu lesson for student

### CODE REVIEW

File: src/components/DropList_SV/BTapChuong.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+'use client';
+import { classNames } from '@/components/classNames';
+import { useRouter } from 'next/navigation';
+
+export default function ExerciseComp({
+  ExerciseTitle,
+  url
```
Comment (leetoneee) [inline src/components/DropList_SV/BTapChuong.tsx]: href

File: src/components/DropList_SV/BTapChuong.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}: {
+  ExerciseTitle: string;
+  url: string;
```
Comment (leetoneee) [inline src/components/DropList_SV/BTapChuong.tsx]: href: string

File: src/components/DropList_SV/BTapChuong.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            strokeWidth="1.5"
+            stroke="currentColor"
+            className={classNames('size-4')}
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              d="M13.5 4.5 21 12m0 0-7.5 7.5M21 12H3"
+            />
+          </svg>
```
Comment (leetoneee) [inline src/components/DropList_SV/BTapChuong.tsx]: use Heroicons libs

File: src/components/DropList_SV/Chuong.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+              strokeWidth="2"
+              stroke="currentColor"
+              className={classNames('size-6')}
+            >
+              <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                d="M2.25 12.75V12A2.25 2.25 0 0 1 4.5 9.75h15A2.25 2.25 0 0 1 21.75 12v.75m-8.69-6.44-2.12-2.12a1.5 1.5 0 0 0-1.061-.44H4.5A2.25 2.25 0 0 0 2.25 6v12a2.25 2.25 0 0 0 2.25 2.25h15A2.25 2.25 0 0 0 21.75 18V9a2.25 2.25 0 0 0-2.25-2.25h-5.379a1.5 1.5 0 0 1-1.06-.44Z"
+              />
+            </svg>
```
Comment (leetoneee) [inline src/components/DropList_SV/Chuong.tsx]: use Heroicons libs

File: src/components/DropList_SV/BTapChuong.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
+'use client';
+import { classNames } from '@/components/classNames';
```
Comment (leetoneee) [inline src/components/DropList_SV/BTapChuong.tsx]: import wrong

File: src/components/DropList_SV/page.tsx
Comment (leetoneee) [inline src/components/DropList_SV/page.tsx:1]: Change filename to DropListSV.tsx

File: src/components/DropList_SV/BTapChuong.tsx (hunk 1 above)
Comment (Binkeurss) [reply src/components/DropList_SV/BTapChuong.tsx]: Done

File: src/components/DropList_SV/BTapChuong.tsx (hunk 2 above)
Comment (Binkeurss) [reply src/components/DropList_SV/BTapChuong.tsx]: Done

File: src/components/DropList_SV/BTapChuong.tsx (hunk 3 above)
Comment (Binkeurss) [reply src/components/DropList_SV/BTapChuong.tsx]: Done

File: src/components/DropList_SV/BTapChuong.tsx (hunk 5 above)
Comment (Binkeurss) [reply src/components/DropList_SV/BTapChuong.tsx]: Done

File: src/components/DropList_SV/page.tsx
Comment (Binkeurss) [reply src/components/DropList_SV/page.tsx:1]: Done

File: src/components/DropList_SV/Chuong.tsx (hunk 4 above)
Comment (Binkeurss) [reply src/components/DropList_SV/Chuong.tsx]: Done

**Comment by Binkeurss** [comment]
fix: update url to href, import Heroicons libs for icon, fix import wrong {classNames}, change filename

**Pull request merged**